packages/paste-core/components/modal/__tests__/index.spec.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import {useUID} from '@twilio-paste/uid-library';
 import {render} from 'react-dom';
-import {render as testRender, fireEvent, cleanup} from '@testing-library/react';
```

Comment:
It looks like we don't have to use cleanup() anymore, could you explain why that is?

Reply:
For sure. The basic answer is that Jest handles this for us. Here's more detail: https://testing-library.com/docs/react-testing-library/api/#cleanup